server/src/option.rs

```diff
@@ -251,3 +261,21 @@ where
         "http".to_string()
     }
 }
+
+pub(self) mod validation {
+    use std::path::PathBuf;
+
+    pub fn file_path(s: &str) -> Result<PathBuf, String> {
+        if s.is_empty() {
+            return Err("empty path".to_owned());
+        }
+
+        let path = PathBuf::from(s);
+
+        if !path.is_file() {
+            return Err("path specified does not point to an accessible file".to_string());
```

Comment:
Print the path as well, so it is clear which path it is talking about

Reply:
Clap prints the value that was passed. 
```error: Invalid value "dir/" for '--tls-cert-path <path>': path specified does not point to an accessible file```

Reply:
oh nice